### ISSUE TITLE
file include.jodit.js exports css without css vars

### ISSUE THREAD

**Issue by undefinederror** [jodit/jodit-react#251]
Why is `jodit/es5/jodit.min.css` exported rather than the css from **any** of the other folders (es2015, es2018, es2021, es2021.en)??
The one from es5 is the only one with hardcoded colors, as opposed to css vars.

That makes it **much** harder and uglier to customize.

**Comment by Naksik** [comment]
I agree with you. This problem has been around for a long time. This is one of the reasons why we don't use jodit-react on our project, as we need customization of theme colors. It would be cool if fixed that

**Comment by somashekarbrdtscblr** [comment]
> es5

what do you use ?

**Comment by xdan** [comment]
This got fixed along the way — the wrapper now imports jodit/es2021/jodit.min.css, which is fully CSS-variable based, so theming via --jd-* custom properties works out of the box. See #279 for a working custom-theme recipe. Closing.
